src/convivaAnalytics.ts

```diff
@@ -34,22 +35,61 @@ export class ConvivaAnalytics extends NativeInstance<ConvivaAnalyticsConfig> {
     this.isInitialized = true;
     return BitmovinPlayerReactNativeAnalyticsConviva.initWithConfig(
       this.nativeId,
-      this.config?.player.nativeId,
+      this.config?.player?.nativeId,
       this.config?.customerKey,
       this.config?.gatewayUrl,
       this.config?.debugLoggingEnabled ?? false
     );
   };
 
+  /**
+   * Initializes a new conviva tracking session.
+   *
+   *  @warning The integration can only be validated without external session managing. So when using this method we can
+   *  no longer ensure that the session is managed at the correct time. Additional: Since some metadata attributes
+   *  rely on the player's source we can't ensure that all metadata attributes are present at session creation.
+   *  Therefore it is possible that we receive a 'ContentMetadata created late' issue after conviva validation.
+   *
+   *  If no source was loaded (or the itemTitle is missing) and no assetName was set via updateContentMetadata
+   *  this method will throw an error.
```

Comment:
Is this double whitespace intended?

```suggestion
   * @warning The integration can only be validated without external session managing. So when using this method we can
   * no longer ensure that the session is managed at the correct time. Additional: Since some metadata attributes
   * rely on the player's source we can't ensure that all metadata attributes are present at session creation.
   * Therefore it is possible that we receive a 'ContentMetadata created late' issue after conviva validation.
   *
   * If no source was loaded (or the itemTitle is missing) and no assetName was set via updateContentMetadata
   * this method will throw an error.
```

Reply:
No, my bad.